### PR TITLE
Update dependencies and add postgres to travis.

### PR DESCRIPTION
**Pull request by maxpeterson**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,10 +16,15 @@ matrix:
     - env: TOX_ENV=py27-djangopre
     - env: TOX_ENV=py34-djangopre
     - env: TOX_ENV=py35-djangopre
+services:
+  - postgresql
+addons:
+  postgresql: "9.4"
```
Comment (Ian-Foote) [inline .travis.yml:22]: Maybe we should test against the latest version of postgres?

**Comment by maxpeterson** [comment]
@meshy @Ian-Foote any thoughts on dropping py 2.7 support - it was added in #9 but I don't know why.

The 2.7 tests are failing because django-appconf (required by django-imagekit) doesn't seem to be 2.7 compatible.

**Comment by Ian-Foote** [comment]
Support was added 6 years ago. Since python 2 is no longer supported, I think we should drop it ourselves.

File: .travis.yml [hunk 2]
```diff
@@ -4,22 +4,24 @@ env:
...
     - TERM='xterm-256color'
   matrix:
-    - TOX_ENV=py27-django111
-    - TOX_ENV=py34-django111
     - TOX_ENV=py35-django111
-    - TOX_ENV=py27-djangopre
-    - TOX_ENV=py34-djangopre
+    - TOX_ENV=py36-django111
     - TOX_ENV=py35-djangopre
+    - TOX_ENV=py36-djangopre
```
Comment (Ian-Foote) [inline .travis.yml:10]: It's probably worthwhile adding 3.7 and 3.8 now. I'd be happy to drop 3.5 too, since that's security fixes only now.

Comment (Ian-Foote) [reply .travis.yml:10]: 3.5 will be end of life in September: https://devguide.python.org/#status-of-python-branches

**Comment by maxpeterson** [comment]
Thanks @Ian-Foote and @meshy - I will get this merged and see about updating Django.

**Pull request merged**